include/lsst/afw/formatters/Utils.h

```diff
@@ -167,6 +171,16 @@ std::string formatFitsProperties(lsst::daf::base::PropertyList const& prop,
                                  std::set<std::string> const& excludeNames = {});
 int countFitsHeaderCards(lsst::daf::base::PropertySet const& prop);
 
+/**
+ * Encode a std::string as a vector of uint8
+ */
+ndarray::Array<std::uint8_t, 1, 1> stringToBytes(std::string const &str);
+
+/**
+ * Decode a std::string from a vector of uint8 returned by stringToBytes
+ */
+std::string bytesToString(ndarray::Array<std::uint8_t const, 1, 1> bytes);
```

Comment:
Why pass by value?

Reply:
I'll change it. It shouldn't matter because ndarray uses a pointer to the data internally, but it will be more obviously safe if I change it.